webservices/tasks/__init__.py

```diff
@@ -45,8 +46,17 @@ def redis_url():
         'webservices.tasks.legal_docs',
     ),
     CELERYBEAT_SCHEDULE=schedule,
+    acks_late=True,
```

Comment:
Should this setting be [`tasks_acks_late`](http://docs.celeryproject.org/en/latest/userguide/configuration.html#std:setting-task_acks_late) instead?